### ISSUE TITLE
is DkimSigner development dead?

### ISSUE THREAD

**Issue by DependableUser** [Pro/dkim-exchange#376]
Does anyone know if it has this been abandoned?
Longstanding issue has not been fixed and some users don't get answers to their questions so I'm wondering...

TIA for any info

**Comment by PacsoT** [comment]
The 3.4.0 verison's relase notes declare that there is no real need for further updates, unless there is some new issue comes up.
So I think it's not dead. Its complete :)

**Comment by DependableUser** [comment]
I would consider it finished if this: [https://github.com/Pro/dkim-exchange/issues/368](https://github.com/Pro/dkim-exchange/issues/368) got fixed but I guess it is what it is.... :(

**Comment by Deckard99** [comment]
I wouldn't say it's complete, but as said above "it is what it is".
There are still problems with signing OOF or NDRs, there is still missing an option for  Ed25519-SHA256 algorithm aso.

**Comment by rvanmill22** [comment]
Is this still valid for existing exchange 2019 - CU14 just installed - need to ensure support before using this. Still active dev on this?

**Comment by DependableUser** [comment]
> Is this still valid for existing exchange 2019 - CU14 just installed - need to ensure support before using this. Still active dev on this?

I upgraded from CU13 to CU14 and it still works fine, the error keeps registering in the event viewer but it can be ignored

**Comment by BruceFeuillette** [comment]
> I wouldn't say it's complete, but as said above "it is what it is". There are still problems with signing OOF or NDRs, there is still missing an option for Ed25519-SHA256 algorithm aso.

I agree, the NDR problem just came to the surface for us. And it's really annoying. I guess I'll go for a Postfix/Linux solution as a workaround.
